elytron/src/main/java/org/wildfly/extension/elytron/Capabilities.java

```diff
@@ -102,7 +102,7 @@
     static final String KEY_MANAGERS_CAPABILITY = CAPABILITY_BASE + "key-managers";
 
     static final RuntimeCapability<Void> KEY_MANAGERS_RUNTIME_CAPABILITY =  RuntimeCapability
-            .Builder.of(KEY_MANAGERS_CAPABILITY, true, KeyManager[].class)
+            .Builder.of(KEY_MANAGERS_CAPABILITY, true, InjectedValue.class) // InjectedValue<KeyManager[]>
```

Comment:
What if I add special injector type, like `KeyManagersInjector` ? Will it be sufficient solution?